File: app/Models/Monitor.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Monitor extends Model
{
    use HasFactory;

    // 白名单
    protected $fillable = [
        'info',
        'created_at',
        'updated_at',
    ];
}

File: app/models/monitor.go
package models

import (
	"github.com/goravel/framework/database/orm"

	"github.com/TheTNB/panel/v2/pkg/tools"
)

type Monitor struct {
	orm.Model
	Info tools.MonitoringInfo `gorm:"not null;serializer:json" json:"info"`
}
